docs/en_US/Tutorial/HowToLaunchFromPython.rst

```diff
@@ -0,0 +1,103 @@
+**How to Launch from Python**
+=============================
+
+Overview
+--------
+This is a new way to launch experiments since ``nni v2.0``. Before, you need to configure your experiment in a yaml configuration file and launch the experiment through ``nnictl`` command. Now, you can also configure and run your experiment directly in the python file. If you are familiar with python programming, this will undoubtedly bring you more convenience.
+
+How to Use
+----------
+After successfully installing ``nni``, you can write your own experimental configuration and run. You can even use python script to launch multiple experiments as you like! The following three steps are necessary.
+
+..
+
+    Step 1 - Initialize a tuner you want to use
+
+
+.. code-block:: python
+
+    from nni.algorithms.hpo.foo_tuner import FooTuner
```

Comment:
do we really have `FooTuner`? better to use a real and simple tuner as example

Reply:
fix it